packages/sources/cfbenchmarks/src/endpoint/birc.ts

```diff
@@ -35,13 +35,24 @@ export interface ResponseSchema {
   }[]
 }
 
+// Tenor must be between -1 and 1
+export const tenorInRange = (tenor: number): boolean => tenor >= -1 && tenor <= 1
+// Using new Date() to get current time in UTC
+export const latestUpdateIsCurrentDay = (timeOfUpdate: number): boolean =>
```

Comment:
I'm scared that this doesn't account for timezone. Wouldn't this cause issues if the tz is different between the DP (London-based?) and the EA instance?

Reply:
I know it's just an error log, but this may cause daily errors for NOPs depending on their location...

Reply:
I had thought than `new Date()` returned the date in UTC already but you may be right. I've updated the method to ensure we compare UTC dates and added some error handling